tests/lib/v9-rule-migration/v9-rule-migration.js

```diff
@@ -0,0 +1,300 @@
+/**
+ * @fileoverview Tests for v9-rule-migration transform.
+ * @author Nitin Kumar
+ * MIT License
+ */
+
+"use strict";
+
+const path = require("path");
+const { applyTransform } = require("@hypermod/utils");
+const assert = require("assert");
+const sinon = require("sinon");
+const { normalizeLineEndings } = require("../../utils");
+const v9MigrationTransform = require("../../../lib/v9-rule-migration/v9-rule-migration");
+
+describe("v9 migration transform", () => {
+    it("should migrate deprecated context methods to new properties", async () => {
```

Comment:
can you also add a test that the vars were defined outside (it's more commonly used to avoid repeating), something like:

```js
module.exports = {
    create(context) {
        const sourceCode = context.getSourceCode();
        const cwd = context.getCwd();
        const filename = context.getFilename();
        const physicalFilename = context.getPhysicalFilename();
        return {
            Program(node) {
                //....
            },
        };
    }
};

```

Reply:
added